tool_services/pipe/_pipe.h

```diff
@@ -18,6 +18,12 @@
 #ifndef PIPE_RX_BUFFER_SIZE
     #define PIPE_RX_BUFFER_SIZE 2048
 #endif
+#ifdef PIPE_TO_LUOS_BUFFER_SIZE
+    #warning "PIPE_TO_LUOS_BUFFER_SIZE is deprecated, use PIPE_RX_BUFFER_SIZE instead"
```

Comment:
We should add the #error directive after this warning (line 22 & 25) to trig a compilation failure